about/join.rst

```diff
@@ -0,0 +1,78 @@
+Usage model and joining
+=======================
+
+Aalto Scientific Computing operates with a community stakeholder model and is
+administered by the School of Science.  Schools, departments, and
+other units join an contribute resources, and get a fair-share of the
+output.  There are two different components to join:
+
+- HPC: Science-IT.  Get a share of computing resources via the :doc:`Triton
+  computing cluster </triton/index>`.
+- Aalto Research Software Engineers (RSE): Support of the RSE program
+  provides intensive hands-on support and service for research
+  software development.
+
+
+For everyone
+------------
+
+Aalto Scientific Computing gets university-level support already, so
+our computing resources are usable by anyone (with a limited share).
```

Comment:
our computing resources are usable by anyone doing research at Aalto (with a limited share).

_Since "anyone" is not really everyone (e.g. students), I have added "doing research". Should we point to triton usage? Or did you have in mind literally everyone?__